app/forms/support_interface/candidate_account_status.rb

```diff
@@ -0,0 +1,46 @@
+module SupportInterface
+  class CandidateAccountStatus
```

Comment:
Do you think we should rename to `CandidateAccountStatusForm` for consistency?

Reply:
Yes we should. We'll be doing this now

Reply:
Done! ♻️